stylus/cozy-ui/build.styl

```diff
@@ -249,6 +255,12 @@
 .c-nav-link
     @extend $nav-link
 
+.c-nav-icon
```

Comment:
Do you see an example to put for `.c-nav-icon` in the comments above?

Reply:
Not really I haven't use this component yet, but I guess you can get inspiration from the react styleguide.

Reply:
I will keep it out of the comment for now, I use it with the `<Icon/>` react component. We may need to test it with a simple `img` or `svg`

Reply:
Tested with a svg here and it works :)